api/camunda/camunda-openapi.yaml

```diff
@@ -577,6 +578,44 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
+
+  /administration/clock/pin:
+    post:
+      tags:
+        - Administration
+        - Clock control
+      summary: Pin the Zeebe engine’s internal clock to a specific time (experimental)
```

Comment:
change-at-the-source, new: 

This name ends up wrapping to 4 lines in the side navigation due to its length. Is there a more terse `summary` we can give it?

<img width="379" alt="image" src="https://github.com/user-attachments/assets/05ba88c7-8f51-4b1b-b254-4cb42ff7758c">

Reply:
We could go for `Pin engine internal clock`

Reply:
owner: me

Reply:
https://github.com/camunda/camunda/pull/22037